### DISCUSSION TITLE
Is it possible to see which applications are installed in a repository via the API?

### DISCUSSION THREAD

**Discussion by justinwyer** [community/community#33813] [Apps, API and Webhooks]
I've search the API docs and it seems impossible to find which apps are installed in a repository using a personal access token. You are able to install and remove applications from a repository using a PAT and you can get an application using a PAT but there is no way to see if an application is already installed using a PAT?

Perhaps this could be added?

**Comment by AyushShahh** [comment]


**Reply by justinwyer** [reply]
@AyushShahh via the API
